internal/bminventory/inventory.go

```diff
@@ -329,6 +329,32 @@ func (b *bareMetalInventory) formatIgnitionFile(cluster *common.Cluster, params
 	return res, nil
 }
 
+func (b *bareMetalInventory) formatNextStepRunnerCommand(clusterID string, hostID string) (string, *[]string) {
```

Comment:
shouldn't it be inside host or instruction manager?

Reply:
I'm not sure. Inventory is the module that handles registration at the API level in `func (b *bareMetalInventory) RegisterHost(...)`, so I thought it would be the right place to put this. The responsibilities of `host.go` seem different. What do you have in mind?

Reply:
We build all the commands from inside host package so i was thinking that it belongs there as well.

Reply:
I will check if it makes sense to move there.

Reply:
Everything under `host` seems related to the next-step cycle and managing the state machine. Won't it be confusing if I put something that's not part of next-step there?

Reply:
if it's not make sense you can  keep it here

Reply:
host is not just for the next step.
Until now that was the major thing there.
I agree that this should move to host